geth/account/accounts_test.go

```diff
@@ -103,9 +107,381 @@ func TestVerifyAccountPasswordWithAccountBeforeEIP55(t *testing.T) {
 	err = common.ImportTestAccount(keyStoreDir, "test-account3-before-eip55.pk")
 	require.NoError(t, err)
 
-	acctManager := account.NewManager(nil)
+	accManager := NewManager(nil)
 
 	address := gethcommon.HexToAddress(TestConfig.Account3.Address)
-	_, err = acctManager.VerifyAccountPassword(keyStoreDir, address.Hex(), TestConfig.Account3.Password)
+	_, err = accManager.VerifyAccountPassword(keyStoreDir, address.Hex(), TestConfig.Account3.Password)
 	require.NoError(t, err)
 }
+
+var (
+	testErrWhisper  = errors.New("Can't return a whisper service")
+	testErrKeyStore = errors.New("Can't return a key store")
+)
+
+func TestManagerTestSuite(t *testing.T) {
+	nodeManager := newMockNodeManager(t)
+	accManager := NewManager(nodeManager)
+
+	keyStoreDir, err := ioutil.TempDir(os.TempDir(), "accounts")
+	require.NoError(t, err)
+	keyStore := keystore.NewKeyStore(keyStoreDir, keystore.LightScryptN, keystore.LightScryptP)
+	defer os.RemoveAll(keyStoreDir) //nolint: errcheck
+
+	testPassword := "test-password"
+
+	// Initial test - create test account
```

Comment:
In the previous version, I believe it was in a separate test case together with an empty password case. I think it was a better solution.

Reply:
I moved the account creation test to `TestManagerTestSuite` in order to create an account while setting up the test. Otherwise, one has to create an account in the beginning of almost every test function. And I also didn't want it to rely on a predefined account and password but rather have it created dynamically in the beginning of every test execution.

Reply:
I can alternatively rely on `status-go/testing.TestConfig` as in `TestVerifyAccountPassword` if that sounds better.

Reply:
Yeah, I got your point. It's ok to create an account that will be used over and over and store it in `ManagerTestSuite`. So, we can leave it here as it is. Let's continue below on `TestCreateAndRecoverAccount` comments.